src/graphql/typeDefs/dataTypes/communityInput.graphql

```diff
@@ -0,0 +1,48 @@
+"Community Query input, use id OR slug, NOT BOTH"
+input CommunityQueryInput {
+  id: ID
+  slug: Slug
+}
+
+input CommunityCreateInput {
+  "Community's name"
+  name: String!
+
+  "unique community slug."
+  slug: Slug!
+
+  "Community Short Description"
+  description: String!
```

Comment:
by the comments this should be called shortDescription. It's a little confusing with aboutUs being the long and then description.. are we just looking for a short/long?

Reply:
Yes, perhaps all we need is short log. Spit-balling names as building out the `Communities` type. I am good with `shortDescription`, `longDescription` if that is more clear and in line with other types.

Reply:
@csell5 Thinking through this and seeing how things have gone to date, I vote that we only have a `description` field. If something more is needed in the future we'll add it.